src/scripts/notebooks/clinvar-udn-submission-analysis/data/output/clinvar_no_mondo.tsv

```diff
@@ -0,0 +1,423 @@
+pref_name
+ORC3-related disorder
+ADCY2-related disorder
+ZNF331 deletion
+NOTCH1-related disorder
+KDR-related disorder
+KIF5B-related osteogenesis imperfecta syndrome
+PAPSS1-related disorder
+MYCBP2-related disorder
+TMEM161B-related lissencephaly
+DHX9-related disorder
+DAGLA-related disorder
+SPTSSA-related disorder|Spastic paraplegia 90A, autosomal dominant
+ZNF865-related disorder
+SREBF2-related disorder
+Neurodevelopmental delay|Seizure
```

Comment:
What does it mean if there are two values?

Reply:
I'm not entirely sure and left for further curator review if/when this moves forward.